libshortfin/src/shortfin/local/systems/host.cc

```diff
@@ -26,6 +26,16 @@ HostCPUSystemBuilder::Deps::Deps(iree_allocator_t host_allocator) {
   iree_task_executor_options_initialize(&task_executor_options);
   iree_hal_task_device_params_initialize(&task_params);
   iree_task_topology_initialize(&task_topology_options);
+
+#ifndef NDEBUG
```

Comment:
good idea

Reply:
I usually hate those kind of add-ons to NDEBUG, but for us, this really is like an assert so feel ok about it.